### PR TITLE
Update php-centos-ubuntu.md

### PR DESCRIPTION
**Pull request by dasharath-wagento**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will...

## Additional information

List all affected URLs 

- ...
- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3507) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3507) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
@dasharath-wagento thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by camdixon** [comment]
Typo I think on line 282 of his commit.   It states PHP 7.1.7, but the commit is referencing updates to docs regarding PHP 7.2 installation.

**Comment by jeff-matthews** [comment]
@dasharath-wagento, you must complete the PR template.

File: guides/v2.3/install-gde/prereq/php-centos-ubuntu.md [hunk 1]
```diff
@@ -206,6 +206,88 @@ To install PHP 7.1 on Ubuntu 14 or 16:
     {:.bs-callout .bs-callout-info}
     The preceding message confirms that the <code>Zend OPcache</code> is installed. We strongly recommend using the OPcache for performance reasons. If your PHP distribution does not come with the OPcache, see the [PHP OPcache documentation](http://php.net/manual/en/opcache.setup.php){:target="_blank"}.
 
+3.	Verify that all [required PHP extensions]({{ page.baseurl }}/install-gde/system-requirements-tech.html#required-php-extensions) were installed:
```
Comment (jeff-matthews) [inline guides/v2.3/install-gde/prereq/php-centos-ubuntu.md]: This is a duplicate of the next step.

**Pull request closed without merging**